frontend/.env.prod.sample

```diff
@@ -0,0 +1,4 @@
+VITE_HOST=localhost # not sure if this is mondey.de in prod?
+VITE_PORT=5173 # not sure if this is 443/80 in prod (or we use nginx or something to direct traffic on 443 to port 5173?)
+VITE_API_PROXY_URL=http://localhost:8000 # I think this is still local host in production because we are directing these from the frontend containers vite proxy running on the machine through the network docker compose has to the backend
+VITE_MONDEY_API_URL=https://mondey.de/api # but this needs to reference the production site since this is where the vite proxy server is running.
```

Comment:
We don't need a frontend .env for production - `VITE_MONDEY_API_URL` is hard-coded to `/api` in the docker-compose, and the rest of these variables are only used by the vite dev server